Gemfile

```diff
@@ -252,8 +252,8 @@ unless ENV["APPLIANCE"]
     gem "factory_bot",      "~>4.11.1", :require => false
     gem "faker",            "~>1.8",    :require => false
     gem "timecop",          "~>0.7.3",  :require => false
-    gem "vcr",              "~>3.0.2",  :require => false
-    gem "webmock",          "~>3.6.0",  :require => false
+    gem "vcr",              "~>5.0.0",  :require => false
+    gem "webmock",          "~>3.7.5",  :require => false
```

Comment:
Should we be more optimistic here?

Reply:
Yeah maybe `"~>5.0"` and `"~>3.7"`?  We can always lock it down if we have an issue in the future while we re-record the cassettes.

Reply:
I agree, I'd rather travis detect incompatible cassettes with new versions as soon as they come out.  Pushed.